SPECS/e2fsprogs/e2fsprogs.spec

```diff
@@ -138,7 +139,7 @@ make -k check |& tee %{_specdir}/%{name}-check-log || %{nocheck}
 %{_libdir}/libe2p.a
 %{_libdir}/libquota.a
 %{_libdir}/libext2fs.a
-%{_libdir}/libss.so
```

Comment:
Unversioned .so files should go to devel packages.

For example, if your package contains: 

   /usr/lib/libfoo.so.3.0.0
   /usr/lib/libfoo.so.3
   /usr/lib/libfoo.so

The versioned shared library files (/usr/lib/libfoo.so.3.2.0 and /usr/lib/libfoo.so.3) are necessary for users to run programs linked against libfoo, so they belong in the base package. The other, unversioned, shared library file (/usr/lib/libfoo.so) is only used to actually link libfoo to code being compiled, and is not necessary to be installed on a users system. This means that it belongs in a -devel package.